cloudinit/distros/alpine.py

```diff
@@ -103,6 +107,20 @@ def _read_hostname(self, filename, default=None):
     def _get_localhost_ip(self):
         return "127.0.1.1"
 
+    def set_keymap(self, layout, model, variant, options):
+        if not layout:
+            LOG.error("Keyboard layout not specified.")
+            return
+        if not variant:
+            LOG.error("Keyboard variant not specified.")
+            return
+        if model:
+            LOG.info("Keyboard model is ignored for Alpine Linux.")
```

Comment:
If somebody passed config expecting it to do something, I think it's worth a warning (same with 2 lines down).

Reply:
done